.github/workflows/beam_LoadTests_Java_GBK_Dataflow_V2_Batch_Java11.yml

```diff
@@ -95,7 +95,7 @@ jobs:
             -PloadTest.mainClass=org.apache.beam.sdk.loadtests.GroupByKeyLoadTest \
             -Prunner=:runners:google-cloud-dataflow-java \
             -Prunner.version=V2 \
-            -PcompileAndRunTestsWithJava11 \
+            -PtestJavaVersion=11 \
```

Comment:
this flag change is to address https://github.com/apache/beam/pull/28726#discussion_r1341443087 (later planned as part of Java21 support: https://github.com/apache/beam/issues/28120#issuecomment-1741033265)